libnetwork/libnetwork_internal_test.go

```diff
@@ -362,39 +362,33 @@ func TestAuxAddresses(t *testing.T) {
 }
 
 func TestUpdateSvcRecord(t *testing.T) {
-	skip.If(t, runtime.GOOS == "windows", "bridge driver and IPv6, only works on linux")
```

Comment:
Looks like this skip is still needed; currently it tries to use `bridge` as driver, which now results in it trying to see if there's a plugin that's to be loaded;


```
=== FAIL: github.com/docker/docker/libnetwork TestUpdateSvcRecord/dual-stack (15.01s)
time="2024-11-13T20:00:39Z" level=info msg="Restoring existing overlay networks from HNS into docker"
time="2024-11-13T20:00:39Z" level=warning msg="Unable to locate plugin: bridge, retrying in 1s"
time="2024-11-13T20:00:40Z" level=warning msg="Unable to locate plugin: bridge, retrying in 2s"
time="2024-11-13T20:00:42Z" level=warning msg="Unable to locate plugin: bridge, retrying in 4s"
time="2024-11-13T20:00:46Z" level=warning msg="Unable to locate plugin: bridge, retrying in 8s"
    libnetwork_internal_test.go:422: assertion failed: error is not nil: could not find plugin bridge in v1 plugin registry: plugin not found
```

Reply:
Ah, yes - thank you ... you played "hunt the red X" before I got to it!